plugins/exporter.koplugin/target/markdown.lua

```diff
@@ -0,0 +1,160 @@
+local _ = require("gettext")
+local UIManager = require("ui/uimanager")
+
+local formatters = {
+    none = "%s",
+    bold = "**%s**",
+    italic = "*%s*",
+    bold_italic = "**_%s_**",
+    underline_markdownit = "++%s++",
+    underline_u_tag = "<u>%s</u>",
+    strikethrough = "~~%s~~",
+}
+
+-- markdown exporter
+local MarkdownExporter = require("base"):new {
+    name = "markdown",
+    extension = "md",
+}
+
+function MarkdownExporter:loadSettings()
```

Comment:
remove. You're using the exact same logic as `BaseExporter`.

Also this function isn't intended to be overriden in subclasses. It should work fine on all plugins. If not we need to refactor the function in the parent class.

Reply:
Ops! Sorry. was toying with that. Forgot to remove.

Removed now.